stdlib/3/multiprocessing/context.pyi

```diff
@@ -131,19 +131,19 @@ class DefaultContext(object):
 
 if sys.platform != 'win32':
     # TODO: type should be BaseProcess once a stub in multiprocessing.process exists
-    class ForkProcess(Any):  # type: ignore
+    class ForkProcess(Any):
```

Comment:
I think these should stay to accommodate the mypy flag that disallows subclassing from Any.